pyroscope/api.go

```diff
@@ -59,6 +60,14 @@ func Start(cfg Config) (*Profiler, error) {
 		return nil, err
 	}
 
+	if cfg.UploadRate == 0 {
+		// For backward compatibility.
+		cfg.UploadRate = 10 * time.Second
+	} else if cfg.UploadRate <= 5*time.Second {
```

Comment:
I would drop the limit, I could see people wanting to do less than 5 seconds.

Reply:
Fixed